### ISSUE TITLE
Liquid withdraw amount error

### ISSUE THREAD

**Issue by AlexLup** [ccxt/ccxt#7902]
Hello, I have found a possible issue with the implementation of withdrawal for Liquid.

- OS: Windows 10
- Programming Language version: Python 3.8
- CCXT version: 1.37.10

Code to reproduce the issue (error received from Liquid: invalid amount)
```
 res = liquid_ex.withdraw(sym='DASH',dest=kraken_dash,qty=0.01)
```

I check and I found that in 'liquid.py' the amount is changed to 0.0  on line 912: 
```
            'amount': self.currency_to_precision(code, amount),
```

As a quick fix, I changed in 
```
            'amount':  amount,
```

I am doing something wrong or it is a bug?

**Comment by kroitor** [comment]
@AlexLup thx for reporting this issue! There was a bug in liquid's fetchCurrencies, we've pushed the fix. Please, update your CCXT to version 1.37.11 (the new build will arrive in 20 minutes). It would be awesome if you report back whether that fixed the issue on your side or not. Thanks again for your feedback!

**Comment by AlexLup** [comment]
> @AlexLup thx for reporting this issue! There was a bug in liquid's fetchCurrencies, we've pushed the fix. Please, update your CCXT to version 1.37.11 (the new build will arrive in 20 minutes). It would be awesome if you report back whether that fixed the issue on your side or not. Thanks again for your feedback!

Problem fixed with the last version: 1.37.12
